### DISCUSSION TITLE
Not able to remove the border of the DatePicker using custom css style

### DISCUSSION THREAD

**Discussion by abhisheknikhra** [wojtekmaj/react-date-picker#630] [Q&A]
I am using following CSS style to remove the border of the DatePicker in ReactJS application, but it is not working. Can you please help:

.datePickerStyle {
  border: none;
}

It comes up like this:

![image](https://github.com/wojtekmaj/react-date-picker/assets/4621522/71daabc1-27bd-4215-9681-687ef82f511f)

**Comment by wojtekmaj** [comment]
Provided that you properly pass `className` to DatePicker component, chances are your CSS is getting ignored because of CSS cascading. Make sure you load default styles first, and custom styles second.

Alternatively, remove default styles altogether, copy its source, and edit it inside your repo instead.
